### DISCUSSION TITLE
Not getting verification email.

### DISCUSSION THREAD

**Discussion by Mrrenzo0861** [community/community#51231] [New to GitHub]
I am not receiving my verification email on my work account.  So I am not able to log into my account.  What can I do because virtual assistance sucks.

**Comment by sumitshinde-84** [comment]
Resetting the password seems to be a solution.

The email should arrive within a few minutes in case of setting a new password, even if you had problems in the authentication process.

**Comment by ramazancetinkaya** [comment]
@Mrrenzo0861, please kindly raise a ticket to [GitHub Support](https://support.github.com/) for the support engineers to investigate the issue.

They usually respond pretty quickly (within a couple of working days).

Hope this clarifies.

If you find this reply useful, please kindly 'Mark as answer' so that others know this issue is already addressed.

Have a good day.
